src/assets/scss/_desktop_navigation.scss

```diff
@@ -88,6 +88,14 @@ $promo-width: 200px;
       background: $colorGrayscale-pebble;
       display: block;
     }
+    > li:nth-of-type(5) {
```

Comment:
"About" is the 5th link on the stage site. https://give-stage2.cru.org/  and that could change in the future.